1-Calling-MSGraph/1-1-AzureAD/AppCreationScripts/AppCreationScripts.md

```diff
@@ -123,8 +123,8 @@ Of course, in real life, you might already get the password as a `SecureString`.
 #### Option 3 (Interactive, but create apps in a specified tenant)
 
   if you want to create the apps in a particular tenant, you can use the following option:
-- open the [Azure portal](https://portal.azure.com)
-- Select the Azure Active directory you are interested in (in the combo-box below your name on the top right of the browser window)
+- open the [Microsoft Entra admin center](https://entra.microsoft.com)
+- Select the Microsoft Entra ID you are interested in (in the combo-box below your name on the top right of the browser window)
```

Comment:
I think this should be 'select the tenant'

Reply:
Great catch. "Tenant" should've been at the end "Azure Active directory" and the script would've caught it. Will fix right now.